spectrumx_visualization_platform/spx_vis/api/views.py

```diff
@@ -124,34 +217,4 @@ class FileViewSet(viewsets.ModelViewSet):
     ordering = ["-created_at"]
 
     def get_queryset(self):
-        """Get the queryset of files for the current user.
-
-        Returns:
-            QuerySet: Filtered queryset containing only the user's files.
-        """
         return File.objects.filter(owner=self.request.user)
-
-    @action(detail=True, methods=["get"])
-    def content(self, request, pk=None):
-        """Get the file content.
-
-        Args:
-            request: The HTTP request
-            pk: The primary key of the file
-
-        Returns:
-            FileResponse: The file content with appropriate content type
-        """
-        file_obj = self.get_object()
-        response = FileResponse(file_obj.file)
-        response["Content-Type"] = file_obj.media_type
-        response["Content-Disposition"] = f'attachment; filename="{file_obj.name}"'
-        return response
```

Comment:
What's the reason for deleting this endpoint? I think this removal breaks a lot of stuff, like capture uploading, and removes the ability to get file contents